mirage/config.ts

```diff
@@ -387,6 +388,8 @@ export default function(this: Server) {
     // Private namespace
     this.namespace = '/_';
 
+    osfResource(this, 'cedar-metadata-template');
```

Comment:
```suggestion
    osfResource(this, 'cedar-metadata-template'. { only }: ['index', 'show']);
```
Since this is a readonly endpoint.